File: python/Product.py
#mengimport Hardware sebagai ortunya (inheritance)
from Hardware import Hardware

class Product(Hardware):
    __Price = 0
    __IdProduct = 0

    def __init__(self, price=0, IdProduct=0):
        self.__Price = price
        self.__IdProduct = IdProduct

    def setPrice(self, Price):
        self.__Price = Price

    def getPrice(self):
        return self.__Price
    
    def setIdProduct(self, IdProduct):
        self.__IdProduct = IdProduct

    def getIdProduct(self):
        return self.__IdProduct

    

File: php/product.php
<?php
    include "hardware.php";

    class product extends hardware{

        private $price = 0;
        private $idProduct = 0;

        public function __construct($price = 0, $idProduct = 0){
            $this->price = $price;
            $this->idProduct = $idProduct;
        }

        public function setPrice($price){
            $this->price = $price;
        }

        public function getPrice(){
            return $this->price;
        }

        public function setIdProduct($idProduct){
            $this->idProduct = $idProduct;
        }

        public function getIdProduct(){
            return $this->idProduct;
        }

        function __destruct(){
        }
    }
?>